srcdeps-maven-plugin/src/test/java/org/srcdeps/mvn/plugin/ExtensionsXmlUtilsTest.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright 2015-2017 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.srcdeps.mvn.plugin;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMResult;
+import javax.xml.transform.stream.StreamSource;
+
+import org.apache.xml.serialize.OutputFormat;
+import org.apache.xml.serialize.XMLSerializer;
+import org.junit.Assert;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+@SuppressWarnings("deprecation")
+public class ExtensionsXmlUtilsTest {
+    private static final String EXTENSIONS_XML_TEMPLATE_REPO = //
```

Comment:
Is there a way to externalize this? Editing XML within a String in Java is usually a pain :)